hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/ClientIds.java

```diff
@@ -167,6 +167,7 @@ private void updateHeartbeat(Path heartbeatFilePath) throws HoodieHeartbeatExcep
           this.fs.create(heartbeatFilePath, true);
       outputStream.close();
     } catch (IOException io) {
+      LOG.error("Unable to generate heartbeat,heartbeatFilePath:{}", heartbeatFilePath, io);
       throw new HoodieHeartbeatException("Unable to generate heartbeat ", io);
     }
```

Comment:
Ditto:

```java
throw new HoodieHeartbeatException("Unable to generate heartbeat for file path " + heartbeatFilePath, io);
```

Reply:
We can remove the log: `LOG.error("Unable to generate heartbeat,heartbeatFilePath:{}", heartbeatFilePath, io);`